praw/reddit.py

```diff
@@ -848,7 +848,7 @@ def put(
         """
         return self._objectify_request(data=data, json=json, method="PUT", path=path)
 
-    def random_subreddit(self, nsfw: bool = False) -> "praw.models.Subreddit":
+    def random_subreddit(self, *, nsfw: bool = False) -> "praw.models.Subreddit":
```

Comment:
This doesn't appear to have the decorator on it.

Reply:
Oops yeah, I get that added.